marshmallow_jsonapi/schema.py

```diff
@@ -207,6 +207,13 @@ def unwrap_request(self, data, many):
 
         data = data['data']
         if many:
+            if not isinstance(data, list):
```

Comment:
To handle more collection types, you can use `marshmallow.utils.is_collection` here.